common/build/eslint-plugin-fluid/src/test/example/no-unchecked-record-access/fileWithOnlyRecordAccess.ts

```diff
@@ -65,6 +85,9 @@ aImplicitType.length; // ok: This should be caught by tsc, not by this custom li
 aImplicitType?.length; // ok: This should be caught by tsc, not by this custom lint rule
 aImplicitType!.length; // ok: This should be caught by tsc, not by this custom lint rule
 
+let aLetExpectingString: string = indexedRecordOfStrings.a; // defect: Assigning index property 'a' to a strict string variable, but 'a' might not be present
+let aLetExpectingStringOrUndefined: string | undefined = indexedRecordOfStrings.a; // ok: Assigning index property 'a' to string or undefined variable, 'a' might not be present
```

Comment:
These are good/fine, but there should also be assignments after the variable declaration. Example:
```typescript
let aLetExpectingString: string;
aLetExpectingString = indexedRecordOfStrings.a;
```